src/extensions/ExtensionManager.as

```diff
@@ -323,6 +329,9 @@ public class ExtensionManager {
 			if(b.isRequester) {
 				if(b.requestState == 2) {
 					b.requestState = 0;
+					// TLF: should it increase the blocking count here instead...?
+					// If so then it'll need also increasing in resetArgsBlockingCount
+					// for any blocks which are requesters.
```

Comment:
I see this as consuming the value that's ready. This could be the last time that the requester is read for a while (or ever!) so there's no need to block the stack until someone tries to read from the "empty" requester again.